### PR TITLE
[v0.4.4] Nginx deployment

### PR DESCRIPTION
**Pull request by LeonardoMariano**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature

* **What is the current behavior?** (You can also link to an open issue here)
We don't have a load balancer for dojot v0.4.4

* **What is the new behavior (if this is a feature change)?**
Nginx will be used as a load balancer

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
Yes, this PR is connected to dojot/dojot#1863 and dojot/dojot#1837

* **Other information**:

### CODE REVIEW

**Review by andersonluisribeiro — CHANGES_REQUESTED** [review]
Good job! I think the docs/vars.md file should be updated as well, as new variables have been added.

File: roles/nginx/defaults/main.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+
+nginx_version: '1.18*'
```
Comment (eduardogmisiuk) [inline roles/nginx/defaults/main.yaml:3]: In the `docs/vars.md` you wrote that the default version is 1.19. Shouldn't it be too?

**Review by eduardogmisiuk — COMMENTED** [review]
Nice! I left only one comment.

**Review by eduardogmisiuk — APPROVED** [review]
LGTM! But don't do a git push force!

**Review by andersonluisribeiro — APPROVED** [review]
:+1:

**Review by mprevide — APPROVED** [review]
It looks ok to me, but I don't know much about nginx, ansible and k8s.

**Pull request merged**